BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.h

```diff
@@ -145,6 +145,37 @@ extern BOXCollaborationRole *const BOXCollaborationRoleViewer;
 extern BOXCollaborationRole *const BOXCollaborationRolePreviewer;
 extern BOXCollaborationRole *const BOXCollaborationRoleUploader;
 
+// Representation Type
+typedef NSString BOXRepresentationType;
+extern BOXRepresentationType *const BOXRepresentationTypeOriginal;
+extern BOXRepresentationType *const BOXRepresentationTypePDF;
+extern BOXRepresentationType *const BOXRepresentationTypeMP4;
+extern BOXRepresentationType *const BOXRepresentationTypeMP3;
+extern BOXRepresentationType *const BOXRepresentationTypePNG;
+extern BOXRepresentationType *const BOXRepresentationTypeJPG;
+extern BOXRepresentationType *const BOXRepresentationType3D;
+extern BOXRepresentationType *const BOXRepresentationTypeFilmstrip;
+extern BOXRepresentationType *const BOXRepresentationTypeDASH;
+extern BOXRepresentationType *const BOXRepresentationTypeHLS;
+extern BOXRepresentationType *const BOXRepresentationTypeCrocodoc;
+extern BOXRepresentationType *const BOXRepresentationTypeDICOM;
+
+// Representation Status
+typedef NSString BOXRepresentationStatus;
+extern BOXRepresentationStatus *const BOXRepresentationStatusSuccess;
+extern BOXRepresentationStatus *const BOXRepresentationStatusPending;
+extern BOXRepresentationStatus *const BOXRepresentationStatusNone;
+
+// Representation content type
+typedef NSString BOXRepresentationContentType;
+extern BOXRepresentationContentType *const BOXRepresentationContentTypeAsset;
+extern BOXRepresentationContentType *const BOXRepresentationContentTypeManifest;
+
+// Representation dimensions
+typedef NSString BOXRepresentationDimensions;
```

Comment:
should the type name be singular? i.e. BOXRepresentationDimension without 's'? since each value contains only 1 dimension?